scripts/archive_leg_to_csv.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+import csv
+import django
+from django import conf
+import click
+from utils import (
+    get_jurisdiction_id,
+)
+
+
+def init_django():  # pragma: no cover
+    conf.settings.configure(
+        conf.global_settings,
+        SECRET_KEY="not-important",
+        DEBUG=False,
+        INSTALLED_APPS=(
+            "django.contrib.contenttypes",
+            "opencivicdata.core.apps.BaseConfig",
+            "opencivicdata.legislative.apps.BaseConfig",
+        ),
+        DATABASES={
+            "default": {
+                "ENGINE": "django.contrib.gis.db.backends.postgis",
+                "NAME": "openstatesorg",
+                "USER": "openstates",
+                "PASSWORD": "openstates",
+                "HOST": "localhost",
+                "PORT": "5405"
+            }
+        },
+        MIDDLEWARE_CLASSES=(),
+    )
+    django.setup()
+
+
+@click.command()
+@click.argument("state_abbr", nargs=-1)
+@click.argument("session", nargs=1)
+def archive_leg_to_csv(state_abbr, session):
+    abbr = state_abbr[0]
+    output_filename = "data/archive_data_legislators/" + abbr + session + "legislators.csv"
+
+    init_django()
+    jurisdiction_id = get_jurisdiction_id(abbr)
+
+    voter_dictionary = {}
+
+    from opencivicdata.legislative.models import Bill, PersonVote
+    bills = Bill.objects.filter(
+        legislative_session__identifier=session,
+        legislative_session__jurisdiction_id=jurisdiction_id).values_list("id", flat=True)
+    for bill in bills:
+        voters = PersonVote.objects.filter(
+            vote_event_id__bill_id=bill
+        )
+        for voter in voters:
```

Comment:
check out collections.Counter for things like this

Reply:
Updated to use Counter